.travis.yml

```diff
@@ -0,0 +1,26 @@
+---
+language: python
+
+matrix:
+  include:
+    - os: linux
+      python: 3.6
+      install:
+        - curl -o bootstrap-salt.sh -L https://bootstrap.saltstack.com
+        - sudo sh bootstrap-salt.sh
+        - sudo mkdir -p /srv/salt
+        - sudo cp -r "amazon-inspector" /srv/salt/
+      script:
+        - sudo salt-call --local --retcode-passthrough state.show_sls amazon-inspector
+        - sudo salt-call --local --retcode-passthrough state.sls amazon-inspector mock=True 
+    - os: windows
+      language: sh
+      python: 3.7
+      before_install:
+        - choco install saltminion
+        - export PATH="/c/salt:$PATH"
+      install:
+        - powershell -ExecutionPolicy RemoteSigned ./build/install_windows.ps1
```

Comment:
We ought to be able to just use `cp` via git bash to copy the files...

```
cp -r "amazon-inspector" /c/salt/srv/salt
```

Reply:
Done.